applications/nrf_desktop/configuration/nrf52840dk_nrf52840/child_image/b0/prj_fast_pair.conf

```diff
@@ -0,0 +1,37 @@
+#
```

Comment:
let's not add another configuration - we can either incorporate the sample usage into default config or wwcb. I think it would be easier if we actually add it to default one along with the wwcb part.

Reply:
That one may cause some issues:

- When building with Fast Pair support, user needs his own Google developer account, Model ID and Anti spoofing key. Currently there are no public Model ID and Anti spoofing key available, so that customers will no longer be able to build such configurations out of the box.
- We will break backwards compatibility (we need a separate partition for Fast Pair provisioning data; the partition must be also populated with data from Google)

If we agree on enabling direct advertising, we may add wwcb compatibility to default configurations in nRF Desktop (we may introduce timeout after which we switch back to undirected advertising in CAF ble_adv to ensure that we will still reconnect to existing hosts at some point). Then fast_pair configuration could then be added as a separate one.